wiremock/mappings/ProbationOffenderSearchApi_SearchByNomsNumber.json

```diff
@@ -0,0 +1,156 @@
+{
+  "id": "05c36ca3-f66c-4ffe-917e-65aff9567af2",
+  "request": {
+    "method": "POST",
+    "urlPathPattern": "/search.*",
+    "bodyPatterns": [
+      {
+          "equalToJson": { "nomsNumber": "A1234AI" }
+      }
+  ]
+  },
+  "response": {
+    "headers": {
+      "Content-Type": "application/json"
+    },
+    "status": 200,
+    "jsonBody": [
+      {
+        "offenderId": 1,
+        "softDeleted": false,
+        "otherIds": {
+          "crn": "X320741",
+          "pncNumber": "2018/0123456X",
+          "croNumber": "SF80/655108T",
+          "nomsNumber": "A1234AI",
+          "previousCrn": "X10001"
```

Comment:
there's also a `"mostRecentPrisonerNumber"` in otherIds which might be something like `"38510A"`